### PR TITLE
virsh_domdirtyrate_calc: Add cases of virsh domdirtyrate-calc

### PR DESCRIPTION
**Pull request by iccaszhulili**
Signed-off-by: Lily Zhu <lizhu@redhat.com>

Case ID: https://polarion.engineering.redhat.com/polarion/#/project/RedHatEnterpriseLinux7/workitem?id=RHEL-279149
Depends on: https://github.com/avocado-framework/avocado-vt/pull/3180

### CODE REVIEW

**Comment by iccaszhulili** [comment]
JOB ID     : 6c876a6c20265f631aafc9b77c23bb11d2fd3972
JOB LOG    : /root/avocado/job-results/job-2021-08-20T04.40-6c876a6/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.domdirtyrate_calc.positive: PASS (69.05 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 69.92 s

File: libvirt/tests/src/virsh_cmd/domain/virsh_domdirtyrate_calc.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        except Exception as e:
+            test.fail("Loading stress failed: %s" % e)
+        session.close()
+
+    def check_output():
+        """
+        Check the dirty rate reported by domstats
+        """
+        res = virsh.domstats(vm_name, "--dirtyrate", debug=True)
+        out_list = res.stdout.strip().splitlines()[1:]
```
Comment (chunfuwen) [inline libvirt/tests/src/virsh_cmd/domain/virsh_domdirtyrate_calc.py:34]: res.stdout-->res..stdout_text ?

**Comment by iccaszhulili** [comment]
@kamilvarga @chunfuwen update, please help to review

**Review by chunfuwen — APPROVED** [review]
LGTM

**Pull request merged**